packages/react/src/analytics/integrations/Guardian/__tests__/Guardian.test.ts

```diff
@@ -0,0 +1,222 @@
+import {
+  EnvironmentTypes,
+  integrations,
+  utils,
+} from '@farfetch/blackout-analytics';
+import { type GuardianIntegrationOptions, GuardianProviders } from '../types';
+import Guardian from '../Guardian';
+
+utils.logger.error = jest.fn();
+utils.logger.warn = jest.fn();
+
+const spyError = jest.spyOn(utils.logger, 'error');
+const spyWarn = jest.spyOn(utils.logger, 'warn');
+
+describe('Guardian', () => {
+  const Forter = require('../providers').Forter;
+  const Riskified = require('../providers').Riskified;
+
+  describe('Guardian integration', () => {
+    it('Should extend the core integration class', () => {
+      expect(Guardian.prototype).toBeInstanceOf(integrations.Integration);
+    });
+
+    it('Should be an integration that loads independently of user consent', () => {
+      expect(Guardian.shouldLoad()).toBe(true);
+      // @ts-expect-error
+      expect(Guardian.shouldLoad({ marketing: true })).toBe(true);
+    });
+  });
+
+  describe('Guardian instance', () => {
+    const spyForter = jest.spyOn(Forter, 'createInstance');
+    const spyRiskified = jest.spyOn(Riskified, 'createInstance');
+
+    beforeEach(jest.clearAllMocks);
+
+    it('should not load all instances by default', () => {
```

Comment:
I think the test description is contrary to what the test does. Can you check, please?